Polyhedron/demo/Polyhedron/Polyhedron_demo_mesh_3_plugin_cgal_code.cpp

```diff
@@ -513,8 +521,7 @@ public Q_SLOTS:
             }
         }
 
```

Comment:
Dummy remark: if  `colors` is now only filled with 0, the for-loop could be replaced by something simpler. Is `colors` used anyway?